### DISCUSSION TITLE
Simple Dark Theme Portrait

### DISCUSSION THREAD

**Discussion by Jojo10011** [mathoudebine/turing-smart-screen-python#276] [Themes]
This is a Portrait Version of my previous [Dark Theme](https://github.com/mathoudebine/turing-smart-screen-python/discussions/195).
![preview](https://github.com/mathoudebine/turing-smart-screen-python/assets/58893218/e0989a90-af85-4272-a946-bda86805f12c)

**Download**
[Landscape-Material-Portrait.zip](https://github.com/mathoudebine/turing-smart-screen-python/files/11941266/Landscape-Material-Portrait.zip)

**Comment by Freekers** [comment]
Thank you very much, looks awesome!

**Comment by Balmmm** [comment]
i used your theme for a long time and i love it but recently i changed my cpu that now has an integrated gpu and i changed my gpu (amd) and i can't see my gpu stats no more. It doesn't work only with your theme. Do you know what can i do to make it work? Thanks
